scripts/microbe_stage/microbe.as

```diff
@@ -589,7 +589,8 @@ class MicrobeSystem : ScriptSystem{
 
             microbeComponent.queuedMovementForce += pos._Orientation * (
                 microbeComponent.movementDirection * force *
-                microbeComponent.movementFactor);
+                microbeComponent.movementFactor *
+                (SimulationParameters::membraneRegistry().getTypeData(microbeComponent.species.membraneType).movementFactor - (microbeComponent.species.membraneRigidity - 0.5) * MEMBRANE_RIGIDITY_MOBILITY_MODIFIER * 2));
```

Comment:
The `* 2` should be put into the constant (MEMBRANE_RIGIDITY_MOBILITY_MODIFIER) instead of being here.